pyLSV2/const.py

```diff
@@ -992,3 +992,471 @@ class ChannelType(IntEnum):
     @classmethod
     def has_value(cls, value):
         return value in set(item.value for item in ChannelType)
+
+class ChannelSignal():
```

Comment:
As mentioned in my comment in #41 this class only works for your machine configuration. I tried it with a programming station at this mapping doesn't work.
Either this has to be moved to your application code or it has to be adapted to dynamically assign the signal numbers after reading them.

Reply:
The names also don't conform to the snake_case naming scheme

Reply:
I added another file (pyLSV2/scripts/channel_signals) which can assign the signal number directly to (pyLSV2/pyLSV2/const.py).
Note: This file must be run only once before using the library for the first time.